### DISCUSSION TITLE
[Blog post] Simon Vrachliotis Joins Tailwind Labs!

### DISCUSSION THREAD

**Discussion by adamwathan** [tailwindlabs/tailwindcss#2043] [Show and tell]
We are over the moon to finally share that @simonswiss has officially joined our team to work full-time on building the Tailwind ecosystem and community!

[Read the announcement on our blog](https://blog.tailwindcss.com/simon-vrachliotis-joins-tailwind-labs)

Simon is a true legend in the functional CSS universe and we are so luck to have him on board 💪  Going to do some amazing work together!

**Comment by jesstelford** [comment]
A perfect fit for the Tailwind team!

Congrats @simonswiss on landing your dream job! And congrats @adamwathan for hiring a dream contributor!

**Reply by simonswiss** [reply]
Thanks @jesstelford! ❤️

It's been awesome hanging around you for the past years - you are a rare breed of "the whole package" skills: dev, business, communication, public speaking, humour. I have learned a lot just watching you "operate", and I know we'll keep talking regularly on discord and other platforms!

**Reply by kuddl** [reply]
Congrats @simonswiss.

**Reply by swyxio** [reply]
love the love in this entire thread :)

**Reply by simonswiss** [reply]
we love you too @sw-yx ❤️

**Comment by russellhaney007** [comment]
Awesome presentation Simon.

**Reply by simonswiss** [reply]
Thanks Russel! 🙌

**Comment by alex-page** [comment]
Wuhuu! Go @simonswiss 🚀

**Comment by kevinhofmaenner** [comment]
Congratulations! I recently purchased TailwindUI and have been loving the experience so far. Looking forward to updates. I really believe you all have a massive upside coming your way. Keep up the great work!
